HousingManagementSystemApi/Startup.cs

```diff
@@ -76,6 +81,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             });
         }
 
+        private static Container getCosmosClientContainer()
+        {
+            var cosmosClient = new CosmosClient(EnvironmentVariableHelper.GetEnvironmentVariable("COSMOS_ENDPOINT_URL"),
+                EnvironmentVariableHelper.GetEnvironmentVariable("COSMOS_AUTHORIZATION_KEY"));
+
+            return cosmosClient.GetContainer(EnvironmentVariableHelper.GetEnvironmentVariable("COSMOS_DATABASE_ID"),
+                EnvironmentVariableHelper.GetEnvironmentVariable("COSMOS_TENANT_CONTAINER_ID"));
+        }
+
         private static string GetEnvironmentVariable(string name) =>
```

Comment:
This method has been relocated to `EnvironmentVariableHelper` ([here](https://github.com/Newarkandsherwood/HousingManagementSystemApi/pull/13/files#diff-b13b4fcdbabb3b91a4c525d69232225e638dce5c2832dcafee99356faf1811ea)) and should be deleted from here as this no longer  used and is code duplication.